website/content/en/docs/building-operators/ansible/reference/scaffolding.md

```diff
@@ -5,14 +5,131 @@ weight: 20
 ---
 
 After creating a new operator project using
-`operator-sdk new --type ansible`, the project directory has numerous generated folders and files. The following table describes a basic rundown of each generated file/directory.
+` operator-sdk init --plugins ansible.sdk.operatorframework.io/v1`,
```

Comment:
Lets use a command with all the options. As is, this command wouldn't produce some of the files documented below.  
```
operator-sdk init --plugins=ansible.sdk.operatorframework.io/v1 \
      --domain=my.domain \
      --group=apps --version=v1alpha1 --kind=AppService \
      --generate-playbook \
      --generate-role
```

Reply:
For less typing, you can use `operator-sdk init --plugins=ansible ...`

Reply:
Solved. Thx.